### ISSUE TITLE
Update issue.

### ISSUE THREAD

**Issue by philshel1** [pymedusa/Medusa#7442]
Hi, I keep getting the error below. This is a windows 10 install and I'm not very savvy with Linux beyond copy and replace.  Can someone give me an idea of what the problem is and where/how to correct it? 
TYIA

2019-12-03 09:48:51 WARNING  CHECKVERSION :: [fcb5a19] git rev-list --left-right "@{upstream}"...HEAD returned : fatal: not a git repository (or any of the parent directories): .git
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git symbolic-ref -q HEAD returned : fatal: not a git repository (or any of the parent directories): .git
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git rev-parse HEAD returned : fatal: not a git repository (or any of the parent directories): .git
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git config remote.origin.url https://github.com/pymedusa/Medusa returned : fatal: not in a git directory
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git config remote.origin.pushurl https://github.com/pymedusa/Medusa returned : fatal: not in a git directory
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git ls-remote --heads origin returned : fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] git fetch --prune origin returned : fatal: not a git repository (or any of the parent directories): .git
2019-12-03 09:48:50 WARNING  CHECKVERSION :: [fcb5a19] Unable to contact github, can't check for update
2019-12-03 09:48:46 WARNING  MAIN :: [fcb5a19] git describe --tags --abbrev=0 None returned : fatal: not a git repository (or any of the parent directories): .git

**Comment by p0psicles** [comment]
We have an issue template. We don't look at issues, that have ignored it.

**Comment by philshel1** [comment]
> We have an issue template. We don't look at issues, that have ignored it.

That's fine, but do you need to be such a dick about it?  I saw no notice, no stickies, or anything at all about there being a template or I would have used it.

**Comment by p0psicles** [comment]
If you open an issue you get the choice between a bug report or feature request. If you choose bug report your input field has all the info filled in. You can just fill in the template. You choose to empty that field.

**Comment by philshel1** [comment]
I have to disagree with you, I did not get a template that I saw.  The box I filled the text in was just like these comment boxs here.  I did not remove anything.  I am not arguing that I screwed up the report, it's either correct or it's not, what pissed me off is that arrogant, dismissive reply.  It's obvious by my account that I'm not a regular contributor, just tell me that I neglected to use the template and that I would need to resubmit it correctly.  Your here to help, if you don't have the personality for it, than don't moderate.
